src/node.cc

```diff
@@ -1221,7 +1228,7 @@ static ExitCode StartInternal(int argc, char** argv) {
   uv_loop_configure(uv_default_loop(), UV_METRICS_IDLE_TIME);
 
   // --build-snapshot indicates that we are in snapshot building mode.
-  if (per_process::cli_options->build_snapshot) {
+  if (per_process::cli_options->per_isolate->build_snapshot) {
```

Comment:
This isn't introduced by this PR specifically, but I guess it makes less sense now to call it `per_isolate` as it seems to imply that child isolates are supposed to inherit this, but in the case of snapshots at least that's not certain (e.g. users might run a few throw-away workers before finishing the snapshot preparation). I don't think it needs to be changed in this PR though, just thinking out loud..

Reply:
I don’t think `per_isolate` is necessarily a bad name for this, whether or not a child Isolate is building a snapshot is still kind of independent from the parent’s settings, although I would imagine that in common situations the values are going to be equal